docs/mkdocs.yml

```diff
@@ -0,0 +1,82 @@
+site_name: SKNNR Documentation
+repo_url: https://github.com/lemma-osu/scikit-learn-knn-regression
+repo_name: lemma-osu/scikit-learn-knn-regression
+docs_dir: pages/
+
+nav:
+  - Home: index.md
+  - Installation: installation.md
+  - Usage: usage.md
+  - "API Reference":
+    - Estimators:
+      - RawKNNRegressor: api/estimators/raw.md
+      - EuclideanKNNRegressor: api/estimators/euclidean.md
+      - MahalanobisKNNRegressor: api/estimators/mahalanobis.md
+      - GNNRegressor: api/estimators/gnn.md
+      - MSNRegressor: api/estimators/msn.md
+    - Transformers:
+      - StandardScalerWithDOF: api/transformers/standardscalerwithdof.md
+      - MahalanobisTransformer: api/transformers/mahalanobis.md
+      - CCATransformer: api/transformers/cca.md
+      - CCorATransformer: api/transformers/ccora.md
```

Comment:
Not sure how publicly visible we want to make the transformers. I didn't mention them at all in the Usage guide yet. Should we include them in the API reference?

Reply:
Great question!  There may be users that only want the transformer capabilities (without the NN addition) so my first inclination is that we include them in the API reference, but leaving them out of the Usage guide also seems reasonable.  Perhaps it's worth a sentence in the Usage Guide to note that underlying most of the estimators are scikit-learn compatible transformers as well, but not necessarily provide examples?

Reply:
I like the idea of briefly mentioning transformers in relation to estimators in the Usage Guide. I added that with a bare-bones example for `CCA`, but feel free to revise!

Reply:
This looks great to me.  A small nit - should we set the call of `cca.fit_transform(X, y)` to a variable or print it, i.e.
```
transformed = cca.fit_transform(X, y)
print(cca.fit_transform(X, y))
```

Reply:
Yes, good call! I went with printing since that's what we do in most of the other examples.